frontend/src/common/Header/Login.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Will this file have the changes from @saddboys 's Login stuff too?

Reply:
Did you see any additions from his merge?? I don't remember having a conflict other than from newest master

Reply:
It seems like this PR will make a new file called `Login.jsx` in `common/Header/` but there will still be a `Login.jsx` in `pages/Login` which is what David was working on

Reply:
Okay thats wack. I essentially, moved existing pages/Login.jsx to common/Header

Reply:
Ill see what sup

Reply:
Okay i've moved the function made by @saddboys!